### DISCUSSION TITLE
Multi-tenancy support for static sites

### DISCUSSION THREAD

**Discussion by gillzon** [vercel/next.js#19631] [Help]
I wonder if there is a way to use nextjs for SaaS applications with multitenancy. For example the customer creates an account and create a webshop and that webshop needs to be generated in a static manner in order for SEO to work. When customer a visit the webshop(unique domain for all shops) then fetch the right static content based on domain?

Is this possible or do i need to render the shop on the client?

**Comment by eric-burel** [comment]
Hi, you may want to check:
- https://github.com/vercel/next.js/discussions/17260
- https://github.com/vercel/next.js/discussions/20841
- https://github.com/vercel/next.js/discussions/17631
